capz/templates/windows-ci.yaml

```diff
@@ -300,6 +300,7 @@ spec:
         kubeletExtraArgs:
           azure-container-registry-config: /etc/kubernetes/azure.json
           cloud-provider: external
+          feature-gates: ${NODE_FEATURE_GATES:-"HPAContainerMetrics=true"}
```

Comment:
I tried this change out and brought up a CAPZ cluster and I don't see the `--feature-gates` flag being added:
```shell
capi@capz-conf-l1kc85-control-plane-j42mf:/var/lib/kubelet$ cat kubeadm-flags.env 
KUBELET_KUBEADM_ARGS="--azure-container-registry-config=/etc/kubernetes/azure.json --cloud-provider=external --container-runtime-endpoint=unix:///var/run/containerd/containerd.sock --pod-infra-container-image=registry.k8s.io/pause:3.9"
```
I am very puzzled by this.